app/src/main/java/io/homeassistant/companion/android/settings/ssid/SsidSummaryProvider.kt

```diff
@@ -0,0 +1,20 @@
+package io.homeassistant.companion.android.settings.ssid
+
+import androidx.preference.Preference
+import io.homeassistant.companion.android.R
+import io.homeassistant.companion.android.domain.url.UrlUseCase
+import javax.inject.Inject
+import kotlinx.coroutines.runBlocking
+
+class SsidSummaryProvider @Inject constructor(
+    private val urlUseCase: UrlUseCase
+) : Preference.SummaryProvider<Preference> {
+
+    override fun provideSummary(preference: Preference): CharSequence {
+        val ssids = runBlocking { urlUseCase.getHomeWifiSsids() }
```

Comment:
This should be getting the values from the preference, not calling from the `UrlUseCase`.